xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShaderGL.cpp

```diff
@@ -160,6 +169,18 @@ bool BaseYUV2RGBGLSLShader::OnEnabled()
     glUniform1f(m_hGammaDstInv, 1/m_pConvMatrix->GetGammaDst());
   }
 
+  if (m_toneMapping)
+  {
+    float param = 0.5;
+    if (m_hasLightMetadata)
+      param = log10(100) / log10(m_lightMetadata.MaxCLL);
+    else if (m_hasDisplayMetadata && m_displayMetadata.has_luminance)
+      param = log10(100) / log10(m_displayMetadata.max_luminance.num/m_displayMetadata.max_luminance.den);
+
+    glUniform3f(m_hCoefsDst, 0.2126, 0.7175, 0.0722);
```

Comment:
magic numbers? looks like they are represent BT709 color space. I think we must use constants for this.

Reply:
yes, needs some refactor of https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/ConversionMatrix.cpp#L40